### DISCUSSION TITLE
Russia localization made termux

### DISCUSSION THREAD

**Discussion by ExtinctDrift19** [termux/termux-packages#7941] [Ideas]
the developers will translate the language into Russian termux language selection in settings

**Comment by ghost** [comment]
There no plans to bring any translations. That has been asked multiple times.

In short, why translations won't be added:
1. Every translation will double amount of work to keep them up-to-date.
2. Besides the application, packages and such resources as Termux:Wiki are English-only. I've made some translation to Russian on Wiki https://wiki.termux.com/wiki/Termux_Wiki_(%D0%BD%D0%B0_%D1%80%D1%83%D1%81%D1%81%D0%BA%D0%BE%D0%BC), but it is incomplete - take this as example on localization support quality.
3. Packages: Termux environment does not support locales. Fixing this may require porting some functions from GNU libc and possibly patching NDK.

Of course, if there will be people who wish to take care about translations, they could be added. But again they should be done for the most if not every parts. Not for just few labels in the app.

**Reply by ExtinctDrift19** [reply]
thanks
